### PR TITLE
Propagate license

### PR DESCRIPTION
**Pull request by tschmidtb51**
- updates package.json to reflect change in detect-node-es

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "use-sidecar",
-  "version": "1.0.4",
+  "version": "1.0.5",
```
Comment (theKashey) [inline package.json:3]: please do not change version manualy.

**Comment by theKashey** [comment]
published

File: package.json (hunk 1 above)
Comment (tschmidtb51) [reply package.json:3]: Ok.

**Comment by tschmidtb51** [comment]
Thank you.

**Pull request merged**